Currently aws-c-auth fails to build (i enabled the tests because aws-sdk-cpp was failing to build with `undefined reference`)

```
aws-c-auth> [100%] Linking C executable aws-c-auth-tests
aws-c-auth> /nix/store/8qm6sjqa09a03glzmafprpp69k74l4lm-binutils-2.40/bin/ld: CMakeFiles/aws-c-auth-tests.dir/credentials_provider_sts_web_identity_tests.c.o: in function `s_credentials_provider_sts_web_identity_new_destroy_from_cached_config':
aws-c-auth> credentials_provider_sts_web_identity_tests.c:(.text+0x5347): undefined reference to `aws_profile_collection_release'
aws-c-auth> /nix/store/8qm6sjqa09a03glzmafprpp69k74l4lm-binutils-2.40/bin/ld: CMakeFiles/aws-c-auth-tests.dir/credentials_tests.c.o: in function `s_profile_credentials_provider_cached_test':
aws-c-auth> credentials_tests.c:(.text+0x63f7): undefined reference to `aws_profile_collection_release'
aws-c-auth> /nix/store/8qm6sjqa09a03glzmafprpp69k74l4lm-binutils-2.40/bin/ld: credentials_tests.c:(.text+0x6401): undefined reference to `aws_profile_collection_release'
aws-c-auth> /nix/store/8qm6sjqa09a03glzmafprpp69k74l4lm-binutils-2.40/bin/ld: credentials_tests.c:(.text+0x65d8): undefined reference to `aws_profile_collection_release'
aws-c-auth> /nix/store/8qm6sjqa09a03glzmafprpp69k74l4lm-binutils-2.40/bin/ld: ../libaws-c-auth.so.1.0.0: undefined reference to `aws_profile_collection_acquire'
aws-c-auth> collect2: error: ld returned 1 exit status
```

--- a/pkgs/development/libraries/aws-c-auth/default.nix
+++ b/pkgs/development/libraries/aws-c-auth/default.nix
@@ -14,13 +14,13 @@
 
 stdenv.mkDerivation rec {
   pname = "aws-c-auth";
-  version = "0.6.22";
+  version = "0.6.26";
 
   src = fetchFromGitHub {
     owner = "awslabs";
     repo = "aws-c-auth";
     rev = "v${version}";
-    sha256 = "sha256-crqoUXPf+2/bhKvvw6fiKNqozVqczbf+aSlK390/w/Q=";
+    sha256 = "sha256-PvdkTw5JydJT0TbXLB2C9tk4T+ho+fAbaw4jU9m5KuU=";
   };
 
   nativeBuildInputs = [

--- a/pkgs/development/libraries/aws-c-http/default.nix
+++ b/pkgs/development/libraries/aws-c-http/default.nix
@@ -11,13 +11,13 @@
 
 stdenv.mkDerivation rec {
   pname = "aws-c-http";
-  version = "0.7.3";
+  version = "0.7.6";
 
   src = fetchFromGitHub {
     owner = "awslabs";
     repo = "aws-c-http";
     rev = "v${version}";
-    sha256 = "sha256-n4BTiqxFz9eOTgh4o78TN2QgCIUbE2Z4jDq27HNORLo=";
+    sha256 = "sha256-pJGzGbIuz8UJkfmTQEZgXSOMuYixMezNZmgaRlcnmfg=";
   };
 
   nativeBuildInputs = [

--- a/pkgs/development/libraries/aws-c-sdkutils/default.nix
+++ b/pkgs/development/libraries/aws-c-sdkutils/default.nix
@@ -7,13 +7,13 @@
 
 stdenv.mkDerivation rec {
   pname = "aws-c-sdkutils";
-  version = "0.1.7";
+  version = "0.1.8";
 
   src = fetchFromGitHub {
     owner = "awslabs";
     repo = "aws-c-sdkutils";
     rev = "v${version}";
-    sha256 = "sha256-qu/+xYorB+QXP5Ixj5ZFP9ZenVYV6hcmxHnH14DEgrU=";
+    sha256 = "sha256-7aLupTbKC2I7+ylySe1xq3q6YDP9ogLlsWSKBk+jI+Q=";
   };
 
   nativeBuildInputs = [

